ng/src/web/pages/credentials/dialog.js

```diff
@@ -114,206 +118,225 @@ class CredentialsDialog extends React.Component {
+                  name="comment"
+                  grow="1"
+                  value={state.comment}
+                  size="30"
+                  maxLength="400"
+                  onChange={onValueChange}/>
+              </FormGroup>
+
+              <FormGroup title={_('Type')}>
+                <Select
+                  onChange={value => this.handleTypeChange(
+                    value, state.autogenerate, onValueChange)}
+                  disabled={is_edit}
+                  items={typeOptions}
+                  value={state.base}/>
+              </FormGroup>
+
+              <FormGroup title={_('Allow insecure use')}>
+                <YesNoRadio
+                  name="allow_insecure"
+                  value={state.allow_insecure}
+                  onChange={onValueChange}/>
+              </FormGroup>
+
+              <FormGroup
+                title={_('Auto-generate')}
+                condition={(state.base === USERNAME_PASSWORD_CREDENTIAL_TYPE ||
+                  state.base === USERNAME_SSH_KEY_CREDENTIAL_TYPE) && !is_edit}>
+                <YesNoRadio
+                  name="autogenerate"
+                  value={state.autogenerate}
+                  onChange={onValueChange}/>
+              </FormGroup>
+
+              <FormGroup
+                title={_('SNMP Community')}
+                condition={state.base === SNMP_CREDENTIAL_TYPE}>
+                {is_edit &&
+                  <Checkbox
+                    name="change_community"
+                    checked={state.change_community === '1'}
+                    checkedValue="1"
+                    unCheckedValue="0"
+                    title={_('Replace existing SNMP community with')}
+                    onChange={onValueChange}/>
+                }
+                <PasswordField
+                  name="community"
+                  value={state.community}
+                  onChange={onValueChange}/>
+              </FormGroup>
+
+              <FormGroup
+                title={_('Username')}
+                flex
+                condition={
+                  state.base === USERNAME_PASSWORD_CREDENTIAL_TYPE ||
+                  state.base === USERNAME_SSH_KEY_CREDENTIAL_TYPE ||
+                  state.base === SNMP_CREDENTIAL_TYPE
+                }>
+                <TextField
+                  name="credential_login"
+                  value={state.credential_login}
+                  onChange={onValueChange}/>
+              </FormGroup>
+
+              <FormGroup
+                title={_('Password')}
+                condition={state.base === USERNAME_PASSWORD_CREDENTIAL_TYPE ||
+                    state.base === SNMP_CREDENTIAL_TYPE}>
+                <Divider>
+                  {is_edit &&
+                    <Checkbox
+                      name="change_password"
+                      checked={state.change_password === '1'}
+                      checkedValue="1"
+                      unCheckedValue="0"
```

Comment:
Same as above YES_VALUE and NO_VALUE should be used.

Reply:
Should be done for all occurrences of '1', '0', 1 and 0 below too.

Reply:
Done